src/pages/aclmanagement.html

```diff
@@ -0,0 +1,87 @@
+                    <th scope="col" class="subject-column">{{'acl_management.rulestable.column.subject' | translate}}</th>
+                    <th scope="col" class="predicate-column">{{'acl_management.rulestable.column.predicate' | translate}}</th>
+                    <th scope="col" class="object-column">{{'acl_management.rulestable.column.object' | translate}}</th>
+                    <th scope="col" class="context-column">{{'acl_management.rulestable.column.context' | translate}}</th>
+                    <th scope="col" class="role-column">{{'acl_management.rulestable.column.role' | translate}}</th>
+                    <th scope="col" class="policy-column">{{'acl_management.rulestable.column.policy' | translate}}</th>
+                    <th scope="col" class="actions-column">{{'acl_management.rulestable.column.actions' | translate}}</th>
+                </tr>
+                <tr>
+                    <th scope="colgroup" colspan="9" class="toolbar">
+                        <div class="pull-right">
+                            <button ng-click="addRule(0)" class="btn btn-link add-rule-btn"
+                                    gdb-tooltip="{{'acl_management.rulestable.actions.add_rule' | translate}}">
+                                <i class="icon-plus"></i>
+                            </button>
+                        </div>
+                    </th>
+                </tr>
+                </thead>
+                <tbody>
+                <tr ng-hide="!loading">
+                    <td colspan="9">
+                        <div onto-loader-new size="100" style="height: 75vh; display: flex;">
+                        </div>
+                    </td>
+                </tr>
+                <tr ng-repeat="rule in rulesModel.aclRules track by $index" class="acl-rule"
+                    ng-class="{'selected': $index === selectedRule}">
+                    <td>{{$index}}</td>
+                    <td class="reorder-cell">
+                        <button ng-click="moveUp($index)" ng-if="$index > 0" class="btn btn-link move-up-btn"
+                                gdb-tooltip="{{'acl_management.rulestable.actions.move_up' | translate}}">
+                            <i class="fa fa-caret-up"></i>
+                        </button>
+                        <button ng-click="moveDown($index)" ng-if="$index < rulesModel.size() - 1"
+                                class="btn btn-link move-down-btn"
+                                gdb-tooltip="{{'acl_management.rulestable.actions.move_down' | translate}}">
+                            <i class="fa fa-caret-down"></i>
+                        </button>
+                    </td>
+                    <td>{{rule.subject}}</td>
+                    <td>{{rule.predicate}}</td>
+                    <td>{{rule.object}}</td>
+                    <td>{{rule.context}}</td>
+                    <td>{{rule.role}}</td>
+                    <td>{{rule.policy}}</td>
```

Comment:
Shouldn't this also be translated? Or you want all data to be shown as is regardless if it is an enumeration of some kind?

Reply:
Exactly. This is data and is displayed as is. Pavel insisted that we don't want to represent it with some component like a checkbox for example.